services/app/src/components/chessBoard/ChessBoard.jsx

```diff
@@ -11,16 +11,48 @@ const ChessBoard = ({ numberOfRows, numberOfCells }) => {
     })),
   );
 
+  const alphabetArray = [...'abcdefghijklmnopqrstuvwxyz'];
+  const numberArray = Array.from(
+    { length: board.length },
+    (_, i) => i + 1,
+  ).reverse();
+
+  board.push(
+    [...Array(numberOfCells).keys()].map((_tile, tileIndex) => ({
+      xAxis: alphabetArray[tileIndex],
+    })),
+  );
+
+  board.forEach((element, i) => {
```

Comment:
Same as above

Reply:
It looks like you want to add Annotations as additional column and row. This logic should be part of the logic in line 6. Let's catch up to see what it looks like.